### PR TITLE
Tests: UserManager API

### PR DESCRIPTION
**Pull request by hytalo-bassi**

### CODE REVIEW

File: tests/test_user_manager.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
+"""Korone is a simple multipurpose Telegram Bot.
+This is a complete rewrite of PyKorone, refer to
+https://github.com/AmanoTeam/PyKorone and README.md for more
+information."""
+
+import os, sys
+
+sys.path.insert(0, f"{os.getcwd()}/src")   # makes possible to call
```
Comment (victorcebarros) [inline tests/test_user_manager.py]: There is no need for this line, korone should be installed on the local environment and then tested with this module.

File: tests/test_user_manager.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+from pytest import fixture
+
+Database.connect("korone-test.db")
+Database.setup()
+
+
+class TestUserManager:
+    """Tests the management of user in database"""
+    user_manager = UserManager(Database)
```
Comment (victorcebarros) [inline tests/test_user_manager.py]: Perhaps you meant `UserManager(Database())` to pass in an instance of the Database to the UserManager?

**Review by victorcebarros — COMMENTED** [review]
Good start for testing!

**Comment by HitaloM** [comment]
Don't forget to use black and isort (with --profile black) to format your code and keep it in line with the project standards.

**Pull request merged**